examples/grpc-client-oauth2-jwt-bearer-grant-type/grpc_client_oauth2_jwt_bearer_grant_type.md

```diff
@@ -28,10 +28,12 @@ Once you run the command, the `grpc_client_pb.bal` file is generated inside the
 
 3. Create a new `grpc_client_oauth2_jwt_bearer_grant_type.bal` Ballerina file inside the `client` package and add the client implementation.
 
+    >**Tip:** You may need to change the trusted certificate file path in the code below. 
+
    ::: code grpc_client_oauth2_jwt_bearer_grant_type.bal :::
 
 4. Execute the commands below to build and run the `client` package.
 
->**Info:** You may need to change the trusted certificate file path. As a prerequisite, start a sample service secured with OAuth2.
+    >**Tip:** As a prerequisite to running the client, start a sample service secured with OAuth2.
```

Comment:
```suggestion
    >** Info:** As a prerequisite to running the client, start a sample service secured with OAuth2.
```

Reply:
Shall we add the prerequisite as `Info`

Reply:
Same as https://github.com/ballerina-platform/ballerina-distribution/pull/3283#discussion_r990543788.

Reply:
Done.